src/ralph/utils.py

```diff
@@ -131,13 +131,13 @@ def statements_are_equivalent(statement_1: dict, statement_2: dict):
     they may still be equivalent.
     """
     # Check that unmutable fields have the same values
-    fields = ["actor", "verb", "object", "id", "result", "context", "attachements"]
+    fields = {"actor", "verb", "object", "id", "result", "context", "attachments"}
```

Comment:
Should I reflect this change in the Changelog? WDYT?

Reply:
Yes, in the `fixed` section. @wilbrdt ?

Reply:
Then you should also fix the `attachements` parameter in  `test_helpers_assert_statement_get_responses_are_equivalent`...

Reply:
https://github.com/openfun/ralph/blob/a42dbae8c4fd856aca6ac1cd555d381712ed80f6/tests/test_helpers.py#L50

Reply:
Fixed @Leobouloc. I wonder, do you know why these tests are under `test_helpers` and not `test_utils`?

Reply:
`tests/helpers.py` was introdued for functions shared across testing while `src/ralph/utils.py` is for functions shared in the Ralph library. 

The function I linked to tests `assert_statement_get_responses_are_equivalent` which is in `tests/helpers.py`, although it does indirectly test `statements_are_equivalent`.